plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2543,18 +2543,15 @@ private static Map<String, Map<ColumnStatisticType, Block>> createColumnToComput
 
     private static DeltaLakeColumnStatistics createDeltaLakeColumnStatistics(Map<ColumnStatisticType, Block> computedStatistics)
     {
-        OptionalLong totalSize = OptionalLong.empty();
-        if (computedStatistics.containsKey(TOTAL_SIZE_IN_BYTES)) {
-            totalSize = getLongValue(computedStatistics.get(TOTAL_SIZE_IN_BYTES));
-        }
+        OptionalLong totalSize = getLongValue(computedStatistics.get(TOTAL_SIZE_IN_BYTES));
         HyperLogLog ndvSummary = getHyperLogLogForNdv(computedStatistics.get(NUMBER_OF_DISTINCT_VALUES_SUMMARY));
         return DeltaLakeColumnStatistics.create(totalSize, ndvSummary);
     }
 
     private static OptionalLong getLongValue(Block block)
     {
-        if (block.isNull(0)) {
-            return OptionalLong.of(0);
```

Comment:
This looks incorrect, so a change looks like a bugfix.
Any test changes?
if this is a dead code until now, add an explanation to the commit message (`Change interpretation of null block for BIGINT value`)

Also, if this is a fix, start the commit title with a "Fix" (not a hard rule, but makes things easier)